vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/PageSteps.java

```diff
@@ -122,10 +122,17 @@ public void openPage(String pageURL)
      * </ul>
      * <p>
      * @param relativeURL A string value of the relative URL
+     * @deprecated Use step: "Then `$variable1` is $comparisonRule `$variable2`".
```

Comment:
deprecated logs are missing mention of expression and dynamic variables used to get rel url